criu/sigact.c

```diff
@@ -0,0 +1,322 @@
+#include "types.h"
+#include "crtools.h"
+#include "infect.h"
+#include "protobuf.h"
+#include "pstree.h"
+#include "parasite.h"
+#include "namespaces.h"
```

Comment:
Do we really need all these headers here?

Reply:
I forgot to get rid of extra headers, let me resolve it.